src/components/pacbio/PacbioPoolLibraryEdit.vue

```diff
@@ -145,14 +145,21 @@ export default {
     ...mapMutations(['updateLibrary']),
     ...mapActions(['applyTags']),
     hasErrors(attribute) {
-      if (!this.library.errors) {
-        return null
+      //Check any errors exist in library for the given attribute
+      if (!this.library.errors || !this.library.errors[attribute]) {
+        return false
       }
-      return !this.library.errors[attribute]
+      return true
     },
     errorsFor(attribute) {
```

Comment:
It looks like errorsFor and hasErrors are practically the same but the reverse. Also here you can do 
`return this.library?.errors?.[attribute]`

Reply:
'errorsFor' returns the error value itself  if any and 'hasErrors' gives a boolean check and return true/false depending on whether there is an error or not.
So, I have removed hasErrors which can in effect be replicated from errorsFor return value